modules/realvuAnalyticsAdapter.js

```diff
@@ -0,0 +1,42 @@
+// RealVu Analytics Adapter
+import adapter from 'src/AnalyticsAdapter';
+import adaptermanager from 'src/adaptermanager';
+
+var realvuAnalyticsAdapter = adapter({
+  global: 'realVuAnalytics',
+  handler: 'on',
+  analyticsType: 'library',
+  url: '//ac.realvu.net/realvu_boost.js'
+});
+
+realvuAnalyticsAdapter.enableAnalytics = function (config) {
+};
+
+realvuAnalyticsAdapter.track = function (eventType, args) {
+}
+var top1 = window;
+try {
+  var wnd = window;
+  while ((top != top1) && (typeof (wnd.document) != 'undefined')) {
+    top1 = wnd;
+    wnd = wnd.parent;
+  }
+} catch (e) { };
+
+realvuAnalyticsAdapter.inView = function (bid, partner_id) {
```

Comment:
Can you let us know how these two functions `inView` and `queue` will be called ? 
Also add a test page. I tried to test with [hello_world](https://github.com/prebid/Prebid.js/blob/master/integrationExamples/gpt/hello_world.html) page by adding enableAnayltics call but no luck.